src/exercises/2-1-Single_use_case_component/Readme.md

```diff
@@ -4,13 +4,44 @@ In this exercise we’ll try to make a dialog component to ask some information
 
 Your project manager comes to your desk and says that they want to greet each app user by their name. To do that you need to implement a dialog that asks the user their name.
 
+### The result
+
+The result should look like this:
+
+```jsx noeditor
+import Prompt from './final/Prompt';
+const [name, setName] = React.useState('');
+const [isVisible, setIsVisible] = React.useState(false);
+<div
+  style={{
+    border: '1px solid #e8e8e8',
+    borderRadius: 3,
+    padding: 16
+  }}
+>
+  {isVisible && (
+    <Prompt
+      title="The univers asks"
+      message="What’s your name, yo?"
+      defaultValue="Incognito"
+      onSubmit={value => {
+        setName(value);
+        setIsVisible(false);
+      }}
+    />
+  )}
+  <p>Name: {name || 'Incognito'}</p>
+  <button onClick={() => setIsVisible(true)}>Ask name</button>
+</div>;
+```
```

Comment:
Could we do this automatically embedding the final code here as well? I didn't figure it out but now it's a good opportunity to do so.